docs/general/ddl/data-types.rst

```diff
@@ -1281,13 +1290,7 @@ timestamp into UTC prior to insertion. Contrast this with the behavior of
     In the absence of an explicit :ref:`WITH TIME ZONE
     <type-timestamp-with-tz>` or :ref:`WITHOUT TIME ZONE
     <type-timestamp-without-tz>`, CrateDB will interpret ``TIMEZONE`` as an
-    alias for ``TIMESTAMP WITH TIME ZONE``.
-
-    This behaviour does not comply with standard SQL and is incompatible with
-    PostgreSQL. CrateDB 4.0 :ref:`deprecated this alias <v4.0.0-deprecations>`
-    and behavior may change in a future version of CrateDB (see `tracking issue
-    #11491`_). To avoid issues, we recommend that you always specify ``WITH
-    TIME ZONE`` or ``WITHOUT TIME ZONE``.
+    alias for ``TIMESTAMP WITHOUT TIME ZONE``.
```

Comment:
Shall we maybe remove the whole `CAUTION` paragraph now that it behaves as the SQL standard defines?

Reply:
Sounds good to me.